### PR TITLE
Signum name system

### PR DESCRIPTION
**Pull request by ohager**
Proposing a DNS like name resolving standard

### CODE REVIEW

**Comment by frankTheTank72** [comment]
@ohager can you also add sip-47 to the main page (readme) please

**Review by frankTheTank72 — CHANGES_REQUESTED** [review]

File: SIP/sip-47.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+sip: 47
+title: Signum Name System (SNS)
+description: Proposal for standardized way of resolving aliases and custom URIs into URLs 
+status: Draft
+type: Standard
+author: johndoe
```
[the remark this answers is not in the document]
Comment (ohager) [reply SIP/sip-47.md:7]: nope

File: SIP/sip-47.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+4. a circular reference occurs
+
+> Keep in mind that resolving to subdomains takes more time the more subdomains are linked to the alias reference list. Resolving subdomains can cause _O(n)_ requests.
+
+![image](./assets/sip-47/linked-alias-list.png "Linked List")
+
+### Different Schemas
+
+For simplicity - while typing the URI inside the browsers URL bar - the `http` and `https` schemas are acceptable, but this requires the user
+to type one of the following (at this time of writing) non-existing top level domains (TLD): `signum`, `signa`, `sig`, `sns`, `free`, `web3`, `crypto` or using the shortform prefix `$`
```
[the remark this answers is not in the document]
Comment (ohager) [reply SIP/sip-47.md]: .zil is for zilliqua - we should not use that
.bitcoin is not us - when bitcoin, then .btc also 
.888 ??  then we could use .666 also

[the remark this answers is not in the document]
Comment (frankTheTank72) [reply SIP/sip-47.md]: okay good point just copied from unstoppable domains...

new proposal:

.coin 
.wallet
.nft
.dao
.blockchain
.p2p
.sustainable

**Review by frankTheTank72 — APPROVED** [review]

**Pull request merged**